### PR TITLE
MA UP receiver migration pre-work

### PR DESCRIPTION
**Pull request by oslynn**
This PR MA receiver migration pre-work.

Test Steps:
1. *Include steps to test these changes*

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

**Review by JessicaWNava — APPROVED** [review]
Looks good to me!

File: prime-router/settings/STLTs/MA/ma-phd.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "MA-MDPH"
+        receivingApplicationOID: "2.16.840.1.113883.19.3.2"
+        receivingFacilityName: "MA-MDPH"
+        receivingFacilityOID: "2.16.840.1.113883.19.3.2"
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB:
```
Comment (JessicaWNava) [inline prime-router/settings/STLTs/MA/ma-phd.yml:30]: Do we keep these settings rather than move them into the receiver transforms?

**Review by JessicaWNava — APPROVED** [review]
Looks good to me!

File: prime-router/settings/STLTs/MA/ma-phd.yml (hunk 1 above)
Comment (oslynn) [reply prime-router/settings/STLTs/MA/ma-phd.yml:30]: No, we don't need to.  Thanks.

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
+
+  - name: ma-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '%{PID}-10-3' ]
+
+  - name: ma-suppress-Hl7-Fields
```
Comment (GilmoreA6) [inline prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml:65]: why is this named "ma-suppress-HL7-Fields"? It doesn't appear to be suppressing anything?

**Review by GilmoreA6 — CHANGES_REQUESTED** [review]
Left a comment on element name

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml (hunk 2 above)
Comment (oslynn) [reply prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml:65]: Thank you.  Good catch.

**Comment by oslynn** [comment]
I closed this PR since it is so far behind.  I created the new PR for MA pre-work:  https://github.com/CDCgov/prime-reportstream/pull/14916

**Pull request closed without merging**